src/include/network/routines.rb

```diff
@@ -712,9 +712,14 @@ def SetLinkUp(dev_name)
       Run("ip link set #{dev_name} up")
     end
 
-    def SetLinkDown(dev_name)
-      log.info("Setting link down for interface #{dev_name}")
-      Run("ip link set #{dev_name} down")
+    def SetLinkDown(dev_name, delete = false)
```

Comment:
I know but what is use case for forcing that? When do we need to delete the interface explicitly. In other words I have been able to always rename a device without ```delete``` being forced so far.